irmin-tezos-utils.opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+maintainer:   "thomas@gazagnaire.org"
+authors:      ["Thomas Gazagnaire" "Thomas Leonard"]
```

Comment:
that's probably not the right authors :-)

Reply:
It would seems so, i'll fix that as well